src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1859,6 +1861,12 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
         identity=identity
     )
 
+    if uptime_sla:
+        mc.sku = ManagedClusterSKU(
+            name="Basic",
```

Comment:
There is only `Basic` sku provided currently ? From compatibility concern, is it possible to expose `--sku` parameter instead of `--uptime-sla` in case there there are more skus in the future ?

Reply:
This is per PM request, and as been discussed for a while, and concluded to have this command.

Reply:
okay...